openapi/openapi.yaml

```diff
@@ -1233,6 +1233,303 @@ paths:
+        - name: titleKeywords
+          in: query
+          required: false
+          description: 'Case-insensitive filter for a keyword in the title; will include only ads which contain the given keyword in their title.'
+          schema:
+            type: string
+            example: 'fiets'
+        - name: status
+          in: query
+          required: false
+          description: 'Filters the result set by the ad status. Should be a comma separated list of possible statuses'
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - 'ACTIVE'
+                - 'PAUSED'
+                - 'DELETED'
+                - 'DELETED_BY_CS'
+                - 'SUSPENDED_BY_CS'
+                - 'BUDGET_REACHED'
+                - 'DAILY_BUDGET_REACHED'
+            uniqueItems: true
+          example:
+            - 'ACTIVE'
+            - 'PAUSED'
+        - name: changedSince
+          in: query
+          required: false
+          schema:
+            $ref: './ads/params.yaml#/components/parameters/ChangedSince'
+        - name: startDate
+          in: query
+          required: false
+          description: 'Determines the start date of the period to select ads with activity for.'
+          schema:
+            type: string
+            format: date
+          example: '2022-12-04'
+        - name: endDate
+          in: query
+          required: false
+          description: 'Determines the end date of the period to select ads with activity for. Should represent a date equal to or after the ``startDate``.'
+          schema:
+            type: string
+            format: date
+          example: '2023-06-13'
+        - name: categoryIds
+          in: query
+          required: false
+          description: 'List of category IDs to filter by. Only leaf category id’s are useful, since ads can only be placed in leaf categories.'
+          schema:
+            type: array
+            items:
+              type: integer
+              format: int64
+          example: [ 55500, 1280000 ]
+        - name: campaignId
+          in: query
+          required: false
+          description: 'Campaign to filter ads by.'
+          schema:
+            type: integer
+            format: int64
+          example: 124422
+      responses:
+        '200':
+          description: 'Success'
+          headers:
+            Oauth-Scope:
+              description: 'Oauth scopes that are available with the supplied token'
+              schema:
+                type: string
+                example: 'api_ro api_rw'
+            Oauth-Scope-Required:
+              description: 'The required permission for this particular call'
+              schema:
+                type: string
+                example: 'get_ads'
+          content:
+            application/sellside.ad.histogram-v1+json:
+              schema:
+                type: object
```

Comment:
To make it clear that the values of the map are expected to be int64, worth adding:

```
         additionalProperties:
           type: integer
           format: int64
```

Reply:
ah, ok, cool ty! New to this stuff